pages/api/discover/random.js

```diff
@@ -42,6 +42,10 @@ export async function getRandomProfileApi() {
 
     return JSON.parse(JSON.stringify(profile ? profile[0] : {}));
   } catch (error) {
+    if (serverEnv.NODE_ENV === "development") {
+      logger.warn("Users not loaded. Please visit /api/system/reload?secret=development");
+      return {};
+    }
```

Comment:
I like it but I think many people will not see it. I know you were not keen on my idea of having an alert box on all the pages https://github.com/EddieHubCommunity/LinkFree/pull/7240/files#diff-24e457f204e088d70fd481f7deae6ac750b25c6c2862dd85a3c9cd1786948456

What about an alert just on the homepage?

Reply:
I created this PR after seeing many screenshots of terminals containing the error.
This error message only occurs when data is not loaded and logging as an error confuses new contributors.

The things I would want from an alert:
1. The alert should only be displayed if the users are not loaded
2. The alert should have 0 chance to be displayed in production
3. The alert should have minimal performance on production as there is no way to compile it out on production build.

Reply:
Yep great points 👍 

I think it should be possible with... ?
- alert box on home page with the same condition you have above plus the vercel environment variable
- the home page already has a data check we can use to see if data is loaded

I can put a quick PR together to see if it is a possible

Reply:
I think the combination of the 2 will improve things.
This PR will downgrade the error to a warning & remove the traceback from the teminal.
Your PR will make it obvious on the homepage